curriculum/challenges/english/25-front-end-development/quiz-git/66f1b06a5a5d10cc100af620.md

```diff
@@ -17,439 +17,442 @@ Answer all of the questions below correctly to pass the quiz.
+`git checkout -- alex`
 
 #### --answer--
 
-Placeholder answer
+`git checkout alex`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Choose the correct option:
+Statement 1 - Team members can collaborate without affecting one another's contributions.
+Statement 2 - Developers can submit changes without waiting for the project maintainer to merge them.
 
 #### --distractors--
 
-Placeholder distractor 1
+Statement 1
 
 ---
 
-Placeholder distractor 2
+Statement 2
 
 ---
 
-Placeholder distractor 3
+Neither statement 1 nor statement 2
 
 #### --answer--
 
-Placeholder answer
+Both statement 1 and statement 2
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Where can a contributor make a pull request?
 
 #### --distractors--
 
-Placeholder distractor 1
+GitHub.com
 
 ---
 
-Placeholder distractor 2
+GitHub Codespaces
 
 ---
 
-Placeholder distractor 3
+GitHub Mobile
 
 #### --answer--
 
-Placeholder answer
+All of the options
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why are code reviews important?
 
 #### --distractors--
 
-Placeholder distractor 1
+Promoting secrecy among teams.
 
 ---
 
-Placeholder distractor 2
+Keeping innovation private.
 
 ---
 
-Placeholder distractor 3
+Decreasing effectiveness.
 
 #### --answer--
 
-Placeholder answer
+Coaching new developers.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the file structure required for a GitHub Pages site?
 
 #### --distractors--
 
-Placeholder distractor 1
+All files must be in a src folder.
 
 ---
 
-Placeholder distractor 2
+Only CSS files are allowed in the root.
 
 ---
 
-Placeholder distractor 3
+JavaScript files cannot be included.
 
 #### --answer--
 
-Placeholder answer
+HTML files should be in the root directory.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do I troubleshoot a 404 error on my GitHub Pages site?
 
 #### --distractors--
 
-Placeholder distractor 1
+Clear your browser cache.
 
 ---
 
-Placeholder distractor 2
+Check if the correct branch is published.
 
 ---
 
-Placeholder distractor 3
+Ensure the file names match the links exactly.
 
 #### --answer--
 
-Placeholder answer
+All of the options.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What command do you use to fetch changes from the remote repository?
 
 #### --distractors--
 
-Placeholder distractor 1
+`git pull`
 
 ---
 
-Placeholder distractor 2
+`git sync`
 
 ---
 
-Placeholder distractor 3
+`git update`
 
 #### --answer--
 
-Placeholder answer
+`git fetch`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you ignore all log files in your repository?
 
 #### --distractors--
 
-Placeholder distractor 1
+`log/*`
 
 ---
 
-Placeholder distractor 2
+`logs/*.log`
 
 ---
 
-Placeholder distractor 3
+`**/*.log`
 
 #### --answer--
 
-Placeholder answer
+`*.log`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common branching strategy for managing features in `git`?
```

Comment:
It might be better to use short description of strategies, rather than just _name_ in distractors and answer.